python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -1012,34 +1012,34 @@ def test_dataframe_error_context(self):
             )
 
             # DataFrameQueryContext with pysparkLoggingInfo - and
-            with self.assertRaises(AnalysisException) as pe:
-                df.withColumn("and_invalid_type", df.id & "string").collect()
-            self.check_error(
-                exception=pe.exception,
-                error_class="DATATYPE_MISMATCH.BINARY_OP_WRONG_TYPE",
-                message_parameters={
-                    "inputType": '"BOOLEAN"',
-                    "actualDataType": '"BIGINT"',
-                    "sqlExpr": '"(id AND string)"',
-                },
-                query_context_type=QueryContextType.DataFrame,
-                pyspark_fragment="and",
-            )
+            # with self.assertRaises(AnalysisException) as pe:
```

Comment:
does it mean this PR has regression?

Reply:
I will address them working in the end. Just made a quick PR to get a review for the overall structure for now.